#!/bin/sh /usr/share/dpatch/dpatch-run
## 102-magic-remove-bennet.dpatch by Josh Triplett <josh@freedesktop.org>
##
## DP: Remove Bennets face format, its match is too generic (Closes: #420855).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/images file-4.20/magic/Magdir/images
--- file-4.20.orig/magic/Magdir/images	2007-01-19 19:31:46.000000000 +0000
+++ file-4.20/magic/Magdir/images	2007-04-26 06:24:31.000000000 +0000
@@ -260,7 +260,7 @@
 
 # other images
 0	string	This\ is\ a\ BitMap\ file	Lisp Machine bit-array-file
-0	string		=!!		Bennet Yee's "face" format
+#0	string		=!!		Bennet Yee's "face" format
 
 # From SunOS 5.5.1 "/etc/magic" - appeared right before Sun raster image
 # stuff.
